plugins/discourse-calendar/app/models/discourse_post_event/event.rb

```diff
@@ -492,14 +492,35 @@ def notify_if_new_event
 
     def calculate_next_recurring_date
       timezone_starts_at = original_starts_at.in_time_zone(timezone)
+      dtstart =
+        Time.find_zone(rrule_timezone).local(
```

Comment:
I think you should be able to use `in_time_zone` here too, so `timezone_starts_at.in_time_zone(rrule_timezone)`